tests/e2e/mapper/mapper_suite_test.go

```diff
@@ -0,0 +1,228 @@
+		cmd = exec.Command("cp", sourcepath, destinationpath)
+		err = utils.PrintCombinedOutput(cmd)
+		Expect(err).Should(BeNil())
+
+		//Apply CRD for devicemodel
+		curpath := utils.Getpwd()
+		file := path.Join(curpath, "../../../build/crds/devices/devices_v1alpha1_devicemodel.yaml")
+		body, err := ioutil.ReadFile(file)
+		if err == nil {
+			client := &http.Client{}
+			BodyBuf := bytes.NewReader(body)
+			req, err := http.NewRequest(http.MethodPost, ctx.Cfg.K8SMasterForKubeEdge+CrdHandler, BodyBuf)
+			Expect(err).Should(BeNil())
+			req.Header.Set("Content-Type", "application/yaml")
+			resp, err := client.Do(req)
+			Expect(err).To(BeNil())
+			utils.InfoV6("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
+			Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
+		}
+
+		//Apply CRD for deviceinstance
+		curpath = utils.Getpwd()
+		file = path.Join(curpath, "../../../build/crds/devices/devices_v1alpha1_device.yaml")
+		body, err = ioutil.ReadFile(file)
+		if err == nil {
+			client := &http.Client{}
+			BodyBuf := bytes.NewReader(body)
+			req, err := http.NewRequest(http.MethodPost, ctx.Cfg.K8SMasterForKubeEdge+CrdHandler, BodyBuf)
+			Expect(err).Should(BeNil())
+			req.Header.Set("Content-Type", "application/yaml")
+			resp, err := client.Do(req)
+			Expect(err).To(BeNil())
+			utils.InfoV6("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
+			Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
+		}
+
+		//Run ./edgecontroller binary
+		cmd = exec.Command("sh", "-c", runController)
+		err = utils.PrintCombinedOutput(cmd)
+		time.Sleep(5 * time.Second)
+		Expect(err).Should(BeNil())
+
+		//Register the Edge Node to Master
+		err = utils.RegisterNodeToMaster(nodeName, ctx.Cfg.K8SMasterForKubeEdge+NodeHandler, nodeSelector)
+		Expect(err).Should(BeNil())
+
+		//Run ./edge_core after node registration
+		cmd = exec.Command("sh", "-c", runEdgecore)
+		err = utils.PrintCombinedOutput(cmd)
+		time.Sleep(5 * time.Second)
+
+		//Check node successfully registered or not
+		Eventually(func() string {
+			status := utils.CheckNodeReadyStatus(ctx.Cfg.K8SMasterForKubeEdge+NodeHandler, nodeName)
+			utils.Info("Node Name: %v, Node Status: %v", nodeName, status)
+			return status
+		}, "60s", "4s").Should(Equal("Running"), "Node register to the k8s master is unsuccessfull !!")
+
+		// Adding label to node
+		utils.ApplyLabelToNode(ctx.Cfg.K8SMasterForKubeEdge+NodeHandler+"/"+nodeName, "bluetooth", "true")
+
+		//dockertag
+		cmd = exec.Command("docker", "tag", "bluetooth_mapper:v1.0", "user/bluetooth_mapper:v1.0")
+		err = utils.PrintCombinedOutput(cmd)
+		Expect(err).Should(BeNil())
+
+		//docker login
+		cmd = exec.Command("sh", "-c", "docker login -u user -p password")
+		err = utils.PrintCombinedOutput(cmd)
+		Expect(err).Should(BeNil())
+
+		//docker push
+		cmd = exec.Command("docker", "push", "lidiya92/bluetooth_mapper:v1.0")
```

Comment:
better we use kubeedge repo instead of personal dockerhub accounts.

Reply:
updated from config